src/algorithms/meta/FilterByAssociations.h

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+// Copyright (C) 2024 Simon Gardner
+
+#pragma once
+
+#include <spdlog/spdlog.h>
+#include <algorithms/algorithm.h>
+#include <string>
+#include <string_view>
+
+#include "services/log/Log_service.h"
+#include "algorithms/interfaces/WithPodConfig.h"
+
+namespace eicrecon {
+
```

Comment:
Can we add a short comment explaining what this does in English?

Reply:
Is the comment above sufficient and not too confusing?

Reply:
Yes. That's good now.

Reply:
Thank you